src/toolkit/enrichment.h

```diff
@@ -57,6 +58,14 @@ double FeedQty(double product_qty, const Assays& assays);
 /// whose units match those of the given product
 double TailsQty(double product_qty, const Assays& assays);
 
+/// @param feed_matl pointer to material
+/// @return ratio of (total mass)/(fissile mass)
+/// Can be used to convert amnt of necessary fissile material
+/// (from FeedQty) into necessary amount of raw material when
+/// non-fissile components are present.  Returns 1 when material
+/// is entirely comprised of U-235 + U-238
+double NonFissileMultiplier(cyclus::Material::Ptr matl);
```

Comment:
I don't think this function is doing what it's documentation states. Perhaps we can discuss its purpose and decide the best course of action (rename the function, etc.).